models/openai_model.py

```diff
@@ -533,17 +533,18 @@ async def send_image_request(self, prompt, vary=None) -> tuple[File, list[Any]]:
         )
 
         # Print the filesize of new_im, in mega bytes
-        image_size = os.path.getsize(temp_file.name) / 1000000
+        image_size = os.path.getsize(temp_file.name) / 1048576
+        guild_file_limit = ctx.guild.filesize_limit / 1048576

```

Comment:
```
i=None row=None>:
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/discord/ui/view.py", line 414, in _scheduled_task
    await item.callback(interaction)
  File "/home/gptbot/cogs/image_prompt_optimizer.py", line 183, in callback
    await self.image_service_cog.encapsulated_send(
  File "/home/gptbot/cogs/draw_image_generation.py", line 49, in encapsulated_send
    file, image_urls = await self.model.send_image_request(
  File "/home/gptbot/models/openai_model.py", line 537, in send_image_request
    guild_file_limit = ctx.guild.filesize_limit / 1048576
AttributeError: 'NoneType' object has no attribute 'guild'
```

Getting this error when image requests happen

Reply:
That's odd, when running it in a regular server it runs just fine for me 
![image](https://user-images.githubusercontent.com/15952522/210884407-03ef1e07-b2ec-411b-aec9-e639c2618679.png)

Reply:
@Hikari-Haru Oops, I forgot to mention. It happens when you click the "Draw" button after optimizing a prompt with /dalle optimize